sky/clouds/service_catalog/gcp_catalog.py

```diff
@@ -364,68 +363,119 @@ def list_accelerators(
-                acc_name in _NUM_ACC_TO_MAX_CPU_AND_MEMORY or
-                acc_name in _ACC_INSTANCE_TYPE_DICTS):
-            new_results[acc_name] = acc_info
-    results = new_results
-
-    # Figure out which instance type to use.
-    infos_with_instance_type: List[common.InstanceTypeInfo] = sum(
-        results.values(), [])
-
-    new_infos = defaultdict(list)
-    for info in infos_with_instance_type:
-        assert pd.isna(info.instance_type) and pd.isna(info.memory), info
-        # We show TPU-VM prices here, so no instance type is needed.
-        # Set it as `TPU-VM` to be shown in the table.
-        if info.accelerator_name.startswith('tpu'):
-            new_infos[info.accelerator_name].append(
-                info._replace(instance_type='TPU-VM'))
-            continue
-        vm_types, _ = get_instance_type_for_accelerator(info.accelerator_name,
-                                                        info.accelerator_count,
-                                                        region=region_filter)
-        # The acc name & count in `info` are retrieved from the table so we
-        # could definitely find a column in the original table
-        # Additionally the way get_instance_type_for_accelerator works
-        # we will always get either a specialized instance type
-        # or a default instance type. So we can safely assume that
-        # vm_types is not None.
-        assert vm_types is not None
-        for vm_type in vm_types:
-            df = _df[_df['InstanceType'] == vm_type]
-            cpu_count = df['vCPUs'].iloc[0]
-            memory = df['MemoryGiB'].iloc[0]
-            vm_price = common.get_hourly_cost_impl(_df,
-                                                   vm_type,
-                                                   use_spot=False,
-                                                   region=region_filter,
-                                                   zone=None)
-            vm_spot_price = common.get_hourly_cost_impl(_df,
-                                                        vm_type,
-                                                        use_spot=True,
-                                                        region=region_filter,
-                                                        zone=None)
-            new_infos[info.accelerator_name].append(
-                info._replace(
-                    instance_type=vm_type,
-                    cpu_count=cpu_count,
-                    memory=memory,
-                    # total cost = VM instance + GPU.
-                    price=info.price + vm_price,
-                    spot_price=info.spot_price + vm_spot_price,
-                ))
-    return new_infos
+
+    # This is a simplified version of get_instance_type_for_accelerator, as it
+    # has to be applied to all the entries in acc_host_df, and the input is more
+    # restricted.
+    def _get_host_vm(acc_name: str, acc_count: int, region: str,
```

Comment:
```suggestion
    def _get_host_instance_type(acc_name: str, acc_count: int, region: str,
```
nit: should this be instance type? to discuss